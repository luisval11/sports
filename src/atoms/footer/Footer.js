import React from "react";

export const Footer = () => {
	return (
		<div className="m-3 text-center">
			<p>Author: Luis Valero Garc√≠a | 2021</p>
		</div>
	);
};
