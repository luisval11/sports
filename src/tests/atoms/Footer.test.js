import { mount, shallow } from "enzyme";
import { Footer } from "../../atoms/footer/Footer";

describe("Test on Footer", () => {
	test("should match initial snapshot", () => {
		let wrapper = shallow(<Footer />);
		expect(wrapper).toMatchSnapshot();
	});

    test("should has the text 'Author: Luis Valero García | 2021'", () => {
		let wrapper = mount(<Footer />);
        expect(wrapper.find('p').text().trim()).toBe('Author: Luis Valero García | 2021');
	});
    
});
