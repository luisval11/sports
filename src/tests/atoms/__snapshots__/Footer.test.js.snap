// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Footer should match initial snapshot 1`] = `
<div
  className="m-3 text-center"
>
  <p>
    Author: Luis Valero Garc√≠a | 2021
  </p>
</div>
`;
